zilliqa/src/bin/convert-key.rs

```diff
@@ -18,10 +19,17 @@ fn main() -> Result<()> {
 
     let secret_key = SecretKey::from_hex(&input.secret_key)?;
 
+    let key_bytes = secret_key.as_bytes();
+    let ecdsa_key =
+        k256::ecdsa::SigningKey::from_bytes(GenericArray::from_slice(&key_bytes)).unwrap();
+
+    let tx_pubkey = TransactionPublicKey::Ecdsa(k256::ecdsa::VerifyingKey::from(&ecdsa_key), true);
+
     let output = json!({
-        "public_key": secret_key.node_public_key(),
+        "bls_public_key": secret_key.node_public_key(),
```

Comment:
We need to update `infra/tf/locals.tf` if we're changing these keys. cc @frankmeds in case there are other references.

Reply:
good catch @JamesHinshelwood. It should be updated only in https://github.com/Zilliqa/zq2/blob/secretkey-small-cleanup/infra/tf/locals.tf#L8

Reply:
Great catch, thanks!